packages/react/src/hooks/use-organizations.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from "@tanstack/react-query";
+import { useSlashID } from "./use-slash-id";
+
+export function useOrganizations() {
+  const { isAuthenticated, user } = useSlashID();
+
+  const result = useQuery({
+    queryKey: ["organizations", user?.oid],
```

Comment:
We could include the person ID here too for other cases to segregate the cache entries.